cypress/e2e/tenure.cy.js

```diff
@@ -0,0 +1,162 @@
+        cy.log("Creating new tenure record");
+        const assetModel = generateAsset()
+        const personModel1 = person();
+        const personModel2 = person();
+        const tenureModel = generateTenure({}, assetModel, [personModel1, { isResponsible: false, personTenureType: "Tenant", ...personModel2 }], undefined, "2014-12-31");
+        tenureModel.householdMembers = [];
+
+        addTestRecordToDatabase("TenureInformation", tenureModel)
+        //given & when
+        cy.getTenureFixture().then(({id: tenureId})=> {
+            
+            tenurePage.visit(tenureId);
+
+            //then
+            tenurePage.scannedHistoricTenureRecords().should('not.exist')
+            tenurePage.tenureResidentsContainer().should('exist')
+            tenurePage.tenureResidentsContainer().contains('This tenure has no household members')
+
+        })
+
+    });
+    
+    it('should view household memebers', ()=> {
+        cy.getTenureFixture().then(({ id: tenureId }) => {
+            //Given & when
+            tenurePage.visit(tenureId);
+
+            //then
+            tenurePage.tenureResidentsContainer().should('exist')
+            tenurePage.householdMemberLink().should('have.attr', 'href').and('include', '/person')
+            tenurePage.householdMemberLink().eq(0).click()
+            cy.url().should('include', '/person')
+
+        });
+    });
+
+    it('should navigate to personal details', ()=>{
+        cy.getTenureFixture().then(({ id: tenureId }) => {
+            //Given & when
+            tenurePage.visit(tenureId);
+
+            tenurePage.viewResidentButtonforPerson().click();
+
+            //then
+            cy.url().should('include', '/person')
+        });
+    });
+
+    it('Accessibility Testing', {'tags': '@Accessibility'}, ()=>{
+        cy.getTenureFixture().then(({ id: tenureId }) => {
+            //Given & when
+            tenurePage.visit(tenureId);
+
+            //then
+            cy.checkA11y(null, null, axeTerminalLog, { skipFailures: true });
+
+            function axeTerminalLog(violations) {
+                cy.task(
+                "log",
+                `${violations.length} accessibility violation${
+                    violations.length === 1 ? "" : "s"
+                } ${violations.length === 1 ? "was" : "were"} detected`
+                );
+
+                const violationData = violations.map(
+                ({ id, impact, description, nodes }) => ({
+                    id,
+                    impact,
+                    description,
+                    nodes: nodes.length,
+                })
+                );
+                cy.task("table", violationData);
+            }
+        });
+    })
+
+    smallDevice.forEach((device) => {
+        it('Mobile view - device with smaller screens', {'tags': '@device'}, ()=> {
+            cy.getTenureFixture().then(({ id: tenureId }) => {
+                //Given 
```

Comment:
I think these tests would be better without the given/when/then comments - they're somewhat ambiguous and add overhead